From 38d34d64109863c97bb903f4f84459d4dbc7d843 Mon Sep 17 00:00:00 2001
From: Prabhakar Kushwaha <prabhakar@freescale.com>
Date: Thu, 24 Dec 2015 15:33:25 +0530
Subject: [PATCH 51/74] driver: net: ldpaa: Increase num of buffers for a pool

Management Complex FW 9.0 set the hardware depletion to be 20 buffers
in order to support multiple pools in DPNI. This requires driver to fill
the pool with at least 21 to be able to receive frames.

So, Increase number of buffers for a pool.

Signed-off-by: Prabhakar Kushwaha <prabhakar.kushwaha@nxp.com>
---
 drivers/net/ldpaa_eth/ldpaa_eth.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ldpaa_eth/ldpaa_eth.h b/drivers/net/ldpaa_eth/ldpaa_eth.h
index af41b27..3b16150 100644
--- a/drivers/net/ldpaa_eth/ldpaa_eth.h
+++ b/drivers/net/ldpaa_eth/ldpaa_eth.h
@@ -24,7 +24,7 @@ enum ldpaa_eth_type {
 };
 
 /* Arbitrary values for now, but we'll need to tune */
-#define LDPAA_ETH_NUM_BUFS		(2 * 7)
+#define LDPAA_ETH_NUM_BUFS		(7 * 7)
 #define LDPAA_ETH_REFILL_THRESH		(LDPAA_ETH_NUM_BUFS/2)
 #define LDPAA_ETH_RX_BUFFER_SIZE	2048
 
-- 
2.1.0.27.g96db324

